sys/dev/dpt/dpt_pci.c

```diff
@@ -62,6 +62,15 @@ __FBSDID("$FreeBSD$");
 static int	dpt_pci_probe	(device_t);
 static int	dpt_pci_attach	(device_t);
 
+static struct dpt_dev {
+	uint16_t vendorid;
+	uint16_t deviceid;
+	const char *description;
+} dpt_devs[] = {
+	{ DPT_VENDOR_ID, DPT_DEVICE_ID,
+	 "DPT Caching SCSI RAID Controller"},
+};	
+
 static int
 dpt_pci_probe (device_t dev)
 {
```

Comment:
The probe should be re-written to use the dpt_dev table.

Reply:
I have used dpt_devs table now